recipes/labmaze/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "labmaze" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmind/labmaze/archive/6d7e8f058428025cd4253f1ef6a1ed6618d9b0ea.zip
+  sha256: dbd3e0b96e1b50936846bdc6f8f63d1199ca6319aefa64c0f5c9a5d29858f605
+  patches:
+    - unvendor_deps.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bazel
+    - bazel-toolchain  # [not win]
+
+  host:
+    - pybind11
+    - pybind11-abi
+    - abseil-cpp
```

Comment:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=630409&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=5645

There's linking warnings about libabseil. Are you accidentally static linking?

Reply:
This should be:
```suggestion
    - libabseil
```

Reply:
> https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=630409&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=5645
> 
> There's linking warnings about libabseil. Are you accidentally static linking?

I am using just an header-only part of abseil. I added an ignore of libabseil's run_exports in https://github.com/conda-forge/staged-recipes/pull/21512/commits/d85e453f27bd12a5b3bd9460dde0d3bd93845525  to account for that.